README.md

```diff
@@ -0,0 +1,107 @@
+Run celestia-app chain
+---
+
+## Pre-Requisites 
+### Clone celestia-app repo and install
+```sh
+git clone https://github.com/celestiaorg/celestia-app.git
+
+cd celestia-app 
+
+make install
+
+# check that celestia-appd --help is working properly
+celestia-appd --help
+```
+### Fork/Clone networks repo
+```sh
+git clone https://github.com/<your_github>/networks.git
+```
+
+## Legend: 
+### - Facilitator (F) - orchestrates genesis setup
+### - Others (O) - other participants
+
+## Phase 0: Environment ideas/hints
+If you want to run celestia app chain, please consider using such setups: 
+
+1. One Digital Ocean Droplet on Ubuntu(1 CPU and 1Gb should be enough) and your local Linux-based(Ubuntu prefferably) OS
+2. Two Digital Ocean Droplets on Ubuntu(1 CPU and 1 Gb should be enough)
+3. Docker Ubuntu based images
```

Comment:
Do we really want this to be DO specific? If yes, we should have a sentence in the beginning ala:
"The following instructions assume that you are using Digital Ocean as a cloud provider. Though it should be straightforward to use another provider or even deploy on a local bare-metal server instead."

Reply:
Yes. I initially thought that docker ubuntu can also be placed here. But now I think making it DO specific can ease the story telling part of this guideline. 
I'll use your suggestion 🙏